src/app/creation/pages/creation-list/creation-list.component.html

```diff
@@ -4,7 +4,7 @@ <h1>{{ title }}</h1>
 
   <div class="section-body">
 
-    <div (click)="goToCreation(creation.id)" *ngFor="let creation of creations" class="card">
+    <div (click)="navigateToCreation(creation.id)" *ngFor="let creation of creations" class="card">
```

Comment:
Y'a une raison particulière pour laquelle tu utilises une `<div>` plutôt qu'un lien `<a>` standard ?

Reply:
Je ne sais plus, il faut que je test avec le lien <a>, mais c'était peut-être à cause de la boucle for.
Je test et je vois si c'est ok avec le <a>.
Mais pourquoi cette question ?

Reply:
Ben c'est important de conserver une logique sémantique dans un template, notamment pour des questions d'accessibilité.

Reply:
Pas de soucis pour remplacer la <div> par un <a>, c'est fait.